python/pyspark/sql/tests.py

```diff
@@ -3476,6 +3476,26 @@ def gen_timestamps(id):
             expected = spark_ts_t.fromInternal(spark_ts_t.toInternal(ts_tz))
             self.assertEquals(expected, ts)
 
+    def test_vectorized_udf_check_config(self):
+        from pyspark.sql.functions import pandas_udf, col
+        orig_value = self.spark.conf.get("spark.sql.execution.arrow.maxRecordsPerBatch", None)
+        self.spark.conf.set("spark.sql.execution.arrow.maxRecordsPerBatch", 3)
+        try:
+            df = self.spark.range(10, numPartitions=1)
+
+            @pandas_udf(returnType=LongType())
+            def checkRecordsPerBatch(x):
```

Comment:
nit: `checkRecordsPerBatch` -> `check_records_per_batch`?

Reply:
Thanks! I'll update it.